index.html

```diff
@@ -143,6 +141,22 @@ <h2>About <br> Myself</h2>
 
                 </div>
         </section>
+        <section class="contact-form">
+            <div class="first-container">
+                <h2>Contact me </h2>
+                <p>If you have an application you <br> are interested in developing, <br> a feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/mzboldye" method="post">
+                <ul>
+                    <li><input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Yeremias"  required></li>
+                    <li><input type="email" id="email" name="user_email" placeholder="natajayanj14@gmail.com" required></li>
+                    <li><textarea name="message" id="msg" cols="30" rows="10" maxlength = "500" placeholder="Write your message here"></textarea></li>
```

Comment:
- [ ] Kindly be noticed that you should add the `required` attribute to the textarea as it has to be entered when the user submits.